tests/apps/workflowsapp/Controller.cs

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+namespace DaprDemoActor
+{
+  using Microsoft.AspNetCore.Mvc;
+  using System.Threading.Tasks;
+  using Dapr.Client;
+  using System;
+
+  [ApiController]
+  [Route("/")]
+  public class Controller : ControllerBase
+  {
+    static string httpEndpoint = "http://127.0.0.1:" + Environment.GetEnvironmentVariable("DAPR_HTTP_PORT");
+    static string grpcEndpoint = "http://127.0.0.1:" + Environment.GetEnvironmentVariable("DAPR_GRPC_PORT");
+    private static readonly DaprClient daprClient = new DaprClientBuilder().UseGrpcEndpoint(grpcEndpoint).UseHttpEndpoint(httpEndpoint).Build();
+    [HttpGet("{workflowComponent}/{instanceID}")]
```

Comment:
Nit: newline before this

```suggestion

    [HttpGet("{workflowComponent}/{instanceID}")]
```